### ISSUE TITLE
How the broadcast command works?

### ISSUE THREAD

**Issue by arbe77** [vysheng/tg#426]
I try "broadcast user1+user2 text" but it doesn't work :(

**Comment by luckydonald** [comment]
No plus sign between. 
```broadcast user_one user_two "Message to deliver"```

**Comment by jonnywilliamson** [comment]
The help screen shows a plus. If I get a chance I'll send a pull request...

Only it's gotten very quiet in here recently :-1:

**Comment by luckydonald** [comment]
Yeah, from the not existing activity shown on vysheng's profile, he probably is on vacation...

The plus symbol is regular expression like syntax.
```+``` means one or more.
(```*``` would mean zero or more)
